spec/index.bs

```diff
@@ -828,25 +851,51 @@ dictionary GPUBufferDescriptor : GPUObjectDescriptorBase {
             1. Set the {{GPUBuffer/[[size]]}} slot of |b| to the value of the {{GPUBufferDescriptor/size}} attribute of |descriptor|.
             1. Set the {{GPUBuffer/[[usage]]}} slot of |b| to the value of the {{GPUBufferDescriptor/usage}} attribute of |descriptor|.
             1. Set the {{GPUBuffer/[[state]]}} internal slot of |b| to [=buffer state/unmapped=].
+            1. Set the {{GPUBuffer/[[mapping]]}} internal slot of |b| to `null`.
+            1. Set the {{GPUBuffer/[[mapping_promise]]}} internal slot of |b| to `null`.
             1. Set each byte of |b|'s allocation to zero.
             1. Return |b|.
         </div>
 </dl>
 
+### {{GPUDevice/createBufferMapped(descriptor)|GPUDevice.createBufferMapped(descriptor)}} ### {#GPUDevice-createBufferMapped}
+
+<dl dfn-type="method" dfn-for="GPUDevice">
+    : <dfn>createBufferMapped(descriptor)</dfn>
+    ::
+        <div algorithm="GPUDevice.createBufferMapped(descriptor)">
+            1. If the result of [$validating GPUBufferDescriptor$](this, descriptor) is false:
+
+                1. Record a validation error in the current scope.
+                <!-- TODO(kangz): Once we have a description of the error monad, explain what the error buffer is. -->
+                1. Create an invalid {{GPUBuffer}} and return the result.
+
+            1. Let |b| be a new {{GPUBuffer}} object.
+            1. Let |m| be an {{ArrayBuffer}} of size the {{GPUBuffer/[[size]]}} slot of |b| filled with zeroes.
```

Comment:
Is `filled with zeroes` mandatory?

Reply:
That would be consistent with buffer write mapping, yes.
```suggestion
            1. Let |m| be a zero-filled {{ArrayBuffer}} of size the {{GPUBuffer/[[size]]}} slot of |b|.
```